docs/proposals/test-reports/pedestrian_tracking.md

```diff
@@ -0,0 +1,127 @@
+# Implement function of automatically algorithm performance tests for Sedna Edge-cloud Re-ID
+
+Person Re-ID based on Cloud-edge collaborative architecture is a new example feature that Sedna will soon support, which can continuously recognize, track and search the target person provided by the user in the source video, and push the video containing the search results to the streaming media server. Since each service module flexibly supports the AI model, it is necessary to provide users with the actual performance test of THE AI algorithm to provide a reference for the actual search effect of the system.
+
+## Goals
+
+1. Supports the existing capabilities and runtime of Sedna and integrate into the Re-ID features;
+2. Use open-source data sets to create test reports automatically, need to define metrics, related interface, preprocessing, and etc;
+3. Have detailed guidance documents.
```

Comment:
It is suggested to add a complete subject here, such as "This framework provides xxx", "This feature supports xxx"